system/etc/apache2/sites-available/default-ssl.conf

```diff
@@ -16,7 +16,7 @@
                 SSLCertificateFile      /etc/ssl/certs/apache-server.crt
                 SSLCertificateKeyFile /etc/ssl/private/apache-server.key
 
-                SSLCACertificatePath /etc/apache2/ssl.crt/
+                SSLCACertificateFile /etc/apache2/ssl.crt/ca.crt
                 SSLVerifyClient optional
```

Comment:
Should I add certbot ?

Reply:
Upload the files to the server.  I'll get them in place.  Then we can setup certbot